### PR TITLE
SCP-1952 Marlowe Marketplace Client: Create an empty shell for the marketplace 

### PR DESCRIPTION
**Pull request by hrajchert**
This PR adds a stripped down version of the Marlowe Dashboard to be used as the basis of the marketplace 

Pre-submit checklist:
- Branch
    - [X] Commit sequence broadly makes sense
    - [X] Key commits have useful messages
    - [X] Relevant tickets are mentioned in commit messages
- PR
    - [x] Self-reviewed the diff
    - [X] Useful pull request description
    - [X] Reviewer requested
- If you updated any cabal files or added Haskell packages:
    - [ ] `nix-shell shell.nix --run updateMaterialized` to update the materialized Nix files
    - [ ] Update `hie-*.yaml` files if needed
- If you changed any Haskell files:
    - [ ] `nix-shell shell.nix --run fix-stylish-haskell` to fix any formatting issues
- If you changed any Purescript files:
    - [X] `nix-shell shell.nix --run fix-purty` to fix any formatting issues

Pre-merge checklist:
- [ ] Someone approved it
- [ ] Commits have useful messages
- [ ] Review clarifications made it into the code
- [ ] History is moderately tidy; or going to squash-merge

### CODE REVIEW

File: marlowe-marketplace-client/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+$ spago2nix generate
+```
+
+This will parse the spago configuration and will generate an updated `.nix` file.
+
+**Note**: If the `spago2nix` command is not available make sure you are inside a nix-shell environment or that your lorri session
+is up and running.
+
+
+### Nix
```
Comment (hrajchert) [inline marlowe-marketplace-client/README.md:90]: This part was copy-pasted from the dashboard and I haven't tested it yet. I will do it in a separate PR focused on deploying.

File: marlowe-marketplace-client/plutus-pab.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+dbConfig:
```
Comment (hrajchert) [inline marlowe-marketplace-client/plutus-pab.yaml:1]: Not sure what this file does, I think it's related for deployed so I left it for later.
cc @shmish111

File: marlowe-marketplace-client/src/Css.purs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+module Css
```
Comment (hrajchert) [inline marlowe-marketplace-client/src/Css.purs:1]: Eventually we should move this to web-commons

File: marlowe-marketplace-client/spago.dhall [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  , "node-fs"
+  , "prelude"
+  , "psci-support"
+  , "remotedata"
+  , "servant-support"
+  , "test-unit"
+  , "undefinable"
+  , "unfoldable"
+  , "uuid"
+  , "web-socket"
```
Comment (hrajchert) [inline marlowe-marketplace-client/spago.dhall:23]: I wanted to build this app without web-socket (as we don't use it here), but the web-commons fail. We could later on create a separate shared lib

File: marlowe-marketplace-client/src/AppM.purs [hunk 5]
```diff
@@ -0,0 +1,38 @@
+module AppM where
```
[the remark this answers is not in the document]
Comment (hrajchert) [reply marlowe-marketplace-client/src/AppM.purs:1]: it doesn't hurt to have it either... that way if we add a server for loading the contracts, we can use this to store the connection options

File: marlowe-marketplace-client/tailwind.config.js [hunk 6]
```diff
@@ -0,0 +1,158 @@
+'use strict';
```
[the remark this answers is not in the document]
Comment (hrajchert) [reply marlowe-marketplace-client/tailwind.config.js:1]: yeah, and I made some slight modifications, and have some concern with the custom spacing :/

**Review by shmish111 — APPROVED** [review]
LGTM

**Pull request merged**